lib/passport-configurator.js

```diff
@@ -11,6 +11,7 @@ var g = SG();
 var loopback = require('loopback');
 var passport = require('passport');
 var _ = require('underscore');
+const bytesToUuid = require('uuid/lib/bytesToUuid');
```

Comment:
Please add uuid as a dependency. It's installed by the other dependencies, but we should be explicit.